tests/sparseml/transformers/compression/test_quantization.py

```diff
@@ -162,11 +162,13 @@ def test_quantization_scale_and_zp(self):
             assert math.isclose(o_scale, n_scale, abs_tol=1e-3, rel_tol=1e-3)
             assert o_zp == n_zp
 
+        # skipping this for commit tests on smaller model, TODO: add back in for the
```

Comment:
Not a fan of TODO's in actual code, would be nice if we could remove this and track this internally(maybe via ASANA?)